### PR TITLE
Change build to use dotnet

### LINKED ISSUE #198: Change to build to use dotnet

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes #198

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @ryanbrandenburg \ @victorhurdugaci

**Comment by victorhurdugaci** [comment]
the change looks okay but you have some test failures

File: .gitattributes [hunk 1]
```diff
@@ -48,3 +48,5 @@
 *.fsproj text=auto
 *.dbproj text=auto
 *.sln text=auto eol=crlf
+
+*.sh eol=lf
```
Comment (ryanbrandenburg) [inline .gitattributes]: I assume this was intentional, why do we define the EOL for .sh and not for most others?

Comment (victorhurdugaci) [inline .gitattributes]: sh files cannot be executed if the eol is CRLF

**Comment by ryanbrandenburg** [comment]
Yeah, seems like the application path is either not being set to the correct place or not being created.

File: .gitattributes (hunk 1 above)
Comment (ryanbrandenburg) [inline .gitattributes]: Huh, learn something new every day.

**Comment by pranavkm** [comment]
Nice catch @victorhurdugaci. The path seems incorrect although it seemed to pass locally. Will update

File: NuGet.config [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvolatiledev/api/v3/index.json" />
```
Comment (pranavkm) [inline NuGet.config]: Will undo this before checking it in.

**Comment by victorhurdugaci** [comment]
:+1:

**Pull request closed without merging**